packages/config/config/devices/0x0154/flow_stop.json

```diff
@@ -1,25 +1,24 @@
-// Popp & Co 009501
-// Popp 09501 Flow Stop
+// Popp & Co Flow Stop
+// Version 1 & 2
 {
 	"manufacturer": "Popp & Co",
 	"manufacturerId": "0x0154",
-	"label": "009501",
-	"description": "Popp 09501 Flow Stop",
+	"label": "Flow Stop",
```

Comment:
AFAIK Popp markets these as two different devices with two different labels (Flow Stop / Flow Stop 2). I think we should have two separate files with different labels for them - if the parameters are identical, you can `$import` them into the Flow Stop 2 file.

Reply:
Ideally template the whole folder :)

Reply:
As the saying goes: Learn to walk before you can run 😅 
Lint failed when `description` was removed, I set it to an empty string.